api/v1alpha1/sandbox_types.go

```diff
@@ -123,10 +131,18 @@ type SandboxStatus struct {
 
 	// status conditions array
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// Replicas is the number of actual replicas.
+	Replicas int32 `json:"replicas"`
+
+	// LabelSelector is the label selector for pods.
+	// +optional
+	LabelSelector string `json:"labelSelector,omitempty"`
```

Comment:
nit: just `selector` to align with Workloads API and Scale API term 

```suggestion
	LabelSelector string `json:"selector,omitempty"`
```

Reply:
fixed